armory/eval/evaluator.py

```diff
@@ -24,6 +25,42 @@
 logger = logging.getLogger(__name__)
 
 
+def tmp_output_subdir(retries=10):
+    """
+    Return (<subdir name>, <tmp subdir path>, <output subdir path>)
+
+    retries - number of times to retry folder creation before returning an error
+        if retries < 0, it will retry indefinitely.
```

Comment:
Could you explain a bit why we expect this to need several retries?

Reply:
Since the subfolder name is timestamp based, there could be collisions when simultaneously launching multiple containers (e.g., on an eval platform). Now that I'm using microsecond timestamps for naming, that chance is much smaller. I figured that 10 retries was more than plentiful to ensure no collisions.